testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -302,6 +304,29 @@ def test_non_callable_method(self, universe):
         with pytest.raises(ValueError):
             self._run_Contacts(universe, method=2, stop=2)
 
+    def test_distance_box(self):
+        u = mda.Universe(TPR, XTC)
+        sel1 = "(resname ARG LYS) and (name NH* NZ)"
+        sel2 = "(resname ASP GLU) and (name OE* OD*)"
+        acidic = u.select_atoms(sel2)
+        basic = u.select_atoms(sel1)
```

Comment:
You could focus the test on a particular pair we know is split across the periodic boundary, to make it easier to directly check the distances measured are what we expect given PBC.